plugins/plugin-maven/che-plugin-maven-server/src/main/java/org/eclipse/che/plugin/maven/server/core/MavenWorkspace.java

```diff
@@ -177,7 +177,7 @@ private void removeProjects(List<MavenProject> removed) {
 
   public void update(List<IProject> projects) {
     manager.update(projects, true);
-    runResolve();
+    //    runResolve();
   }
 
   private void runResolve() {
```

Comment:
unused?

Reply:
Disabled not to interfere the introduced changes but can't be removed due to compilation issues.